LDM-652.tex

```diff
@@ -0,0 +1,210 @@
+\item Risk and mitigation plans.
+\end{itemize}
+
+All relevant documentation should be provided to the review committee no later than 2 weeks prior to the date of the review.
+
+\subsection{Applicable Documents}
+\label{sec:docs}
+
+\addtocounter{table}{-1}
+
+\begin{tabular}[htb]{l l}
+\citeds{LSE-319} & LSST Science Platform Vision Document \\
+\citeds{LDM-554} & LSST Science Platform Requirements\\
+\citeds{LDM-542} & LSST Science Platform Design\\
+\citeds{LSE-163} & LSST Data Products Definition Document \\
+\citeds{LPM-17} & LSST Science Requirements Document (SRD) \\
+\citeds{LSE-61}  & LSST DM Subsystem Requirements (DMSR)  \\
+\citeds{LDM-148} & LSST Data Management System Design \\
+\citeds{LSE-30} & LSST Observatory System Specifications (OSS) \\
+\end{tabular}
+
+% references
+\subsection{References\label{sect:references}}
+\renewcommand{\refname}{}
+\bibliography{lsst,refs,books,refs_ads}
+
+%acronyms
+\subsection{Acronyms \label{sect:acronyms}} % include acronyms.tex generated by the generateAcronyms.py (in texmf/scripts)
+\input{acronyms}
+
+
+\newpage
+%%% Charge questions
+\section{Charge to the review committee}
+\label{sec:charge}
+The review committee is asked to assess the following items based on the material presented and made available.
+%
+\begin{enumerate}
+% We need to do some work before the review to get this into shape. Our requirements traceability is not great. Would propose to get the key people together and knock it off in a dedicated session
+\item Is the traceability of requirements from higher design documents, e.g. from \citeds{LPM-17}:The LSST Science Requirements Document,
+to \citeds{LDM-554}:The LSST Science Platform Requirements, complete and will it ensure coverage of the four key LSST science themes?
+%
+\item Are the stakeholders clearly identified and understood? Have the requirements been prioritized and communicated to the stakeholders?
+%
+% Portal in LDM-554 are the portal requirements. DAX reqs less clear - DMTN-090 - more VO centric design. - Well captured.
+\item Does the design presented in \citeds{LDM-542}: LSST Science Platform Design capture the requirements for the LSP as detailed in \citeds{LDM-554}:The LSST Science Platform Requirements?
+%
+\item Are the verification, validation and software quality assurance plans adequate?
+%
+\item Does the performance of the current system and development status inspire confidence that both the interim and operations-era functionality can be met?
+%
+\item How does the design of the LSP compare with that of other contemporary astronomical data archives and interfaces, or, more generally, other scientific data analysis environments? How well does the design and current version reflect trends in software engineering? Do the current design and technology choices give confidence that the LSP can evolve over time with the needs of 21st century astronomy?
```

Comment:
I think "current version" meant the at-time-of-review deployment, e.g., in PDAC and lsst-lspdev?  In the context of "trends in SE", though, I think "implementation" is a bit clearer than "version", if I've captured your intent correctly.

```suggestion
\item How does the design of the LSP compare with that of other contemporary astronomical data archives and interfaces, or, more generally, other scientific data analysis environments? How well does the design and current implementation reflect trends in software engineering? Do the current design and technology choices give confidence that the LSP can evolve over time with the needs of 21st century astronomy?
```